modules/Campaigns/metadata/subpaneldefs.php

```diff
@@ -245,7 +277,11 @@
             'order' => 185,
             'module' => 'CampaignLog',
             'get_subpanel_data' => "function:track_log_entries",
-            'function_parameters' => array(0 => 'blocked', 'EMAIL_MARKETING_ID_VALUE' => '',/*'group_by'=>'campaign_log.target_id','distinct'=>'campaign_log.target_id'*/),
+            'function_parameters' => array(
+                0 => 'blocked',
+                'EMAIL_MARKETING_ID_VALUE' => '',
+                /*'group_by'=>'campaign_log.target_id','distinct'=>'campaign_log.target_id'*/
```

Comment:
Hi @Dillon-Brown could we remove the commented code please? ` /*'group_by'=>'campaign_log.target_id','distinct'=>'campaign_log.target_id'*/`
We should probably remove from the other subpanel definitions on this file.

Reply:
Pushed here and the rest of the file.